spec/requests/users_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  before(:each) do
+    get users_path
+  end
+
+  describe 'GET /users' do
+    it 'should return a list of users' do
+      get '/users'
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'GET /users/:id' do
+    it 'should return a user' do
+      get '/users/1'
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'GET /index' do
```

Comment:
- Good job done so far, but according to the project requirements, you are to test that for each action, the `http_status` is correct, it renders the correct placeholder and template. For the `user#show` action, please test that the controller renders the correct template and placeholder. Please fix this 👍 👍